nautobot/dcim/models/device_components.py

```diff
@@ -35,6 +35,7 @@
 from nautobot.extras.models import (
     ChangeLoggedModel,
     RelationshipModel,
+    RoleField,
```

Comment:
We do not have them yet. But I would consider them to be out of scope for a 3-point story. Especially when the `Interface` model inherits from `CableTermination`, `PathEndpoint`, and etc, and making a factory for it might be tricky.